pkg/migrations/mls/20240109001927_add-messages.up.sql

```diff
@@ -0,0 +1,15 @@
+SET statement_timeout = 0;
+
+--bun:split
+
+CREATE TABLE messages (
+    topic VARCHAR(200) NOT NULL,
+    tid VARCHAR(96) NOT NULL,
```

Comment:
Given how strict the ordering requirements are, I'd really rather this rely on a sequence than the server time. Clocks can drift, and any re-ordering can corrupt the group state with the way MLS works. Even if those conflicts are improbable, it makes debugging a lot easier if they are impossible.

Reply:
Ok agree, that makes sense. I'll update it to do that. 

I'm going to close down this PR as it is, I think I'd also like to just split up the messages by type too; group vs welcome messages, and get rid of the idea of topics completely.